### PR TITLE
Minor fix. And removed images to prevent conflict (IDs vs Names)

### PR DESCRIPTION
**Pull request by NomadCHK**
Refactor the evolution logic to ensure checks are more robust by verifying if the "evolves_from" attribute is not None before performing string comparisons. This prevents potential errors when dealing with incomplete data and streamlines the code for better clarity. Improved readability will help maintain the code in the long run. Also removed old images.

No related issues.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def run_script(self):
             ### BATTLE START
             self.image_processor.check_and_click_until_found(
                 self.template_images["TIME_LIMIT_INDICATOR"],
-                'Time limit indicator',
+                "Time limit indicator",
```
Comment (AdriaGual) [inline bot.py:124]: Related to this issue:
https://github.com/AdriaGual/pokemon-pocket-bot/issues/4

It should be single quotes.

```suggestion
                'Time limit indicator',
```

Comment (NomadCHK) [reply bot.py:124]: ok, sorry. by default pattern my python linter makes all double quotes. I will fix. also have another fix

**Comment by NomadCHK** [comment]
I just submitted more fixes, also did your suggestion.

**Comment by NomadCHK** [comment]
I just noticed that the API is responding with less data. I still have the CSV with complete data here, I'll include it, what do you think?

**Review by AdriaGual — APPROVED** [review]
LGTM

**Pull request merged**